archinstall/lib/disk/partition.py

```diff
@@ -301,6 +302,13 @@ def format(self, filesystem=None, path=None, log_formatting=True, options=[]):
 				raise DiskError(f"Could not format {path} with {filesystem} because: {handle.decode('UTF-8')}")
 			self.filesystem = filesystem
 
+		elif filesystem == 'ntfs':
+			options = ['-f'] + options
+
+			if (handle := SysCommand(f"/usr/bin/mkfs.ntfs {' '.join(options)} {path}")).exit_code != 0:
```

Comment:
I guess we'd want a `-Q` option here as well for quick?

Reply:
That might be worthwhile if it speeds up partitioning, I didn't think that it was taking too long when I was doing it in my virtual machine, but it was only a 35 GB volume.